source/standards/running-services-as-containers.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Running services as containers
+expires: 2018-05-02
+---
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+## Principles
+
+This guidance should not be applied to development environments using containers.
+
+Where possible, web applications should use the PaaS as per the [hosting guidance](standards/hosting.html).
+
+It is not recommended to run bare containers on the PaaS, as provided by the [documentation](https://docs.cloud.service.gov.uk/#deploy-a-docker-image-experimental).
```

Comment:
I wonder if the link should be on "not recommended" and lose the "as provided by the [documentation]"

maybe even make it.
Running bare containers on the PaaS is [not recommended] by the PaaS team.

Reply:
While the official Cloud Foundry docs do say it is possible, I think currently the PaaS team do not officially support this usage. Running a service without support from the platform is a high risk. If this support changes that this guidance could certainly change.

Reply:
The reason we don't recommend using Docker images on CloudFoundry is that doing so ties the availability of your application to the availability of the Docker registry in use (eg Docker Hub). If the registry is unavailable for any reason, your application could fail because the platform will be unable to start new instances of the application. This does not just apply to new deployments, but any event that would cause the platform to reschedule an instance of the application (eg cell failures, platform deployments etc).